recipes/conf_files.rb

```diff
@@ -32,7 +30,6 @@
   end.uniq
 conf_files_dirs.each do |dir|
   directory "#{node['dovecot']['conf_path']}/#{dir}" do
-    name ::File.join(node['dovecot']['conf_path'], dir)
```

Comment:
The `File.join` is there for different OS compatibility. But yeah, probably should be removed :+1: